changelog.d/502.misc

```diff
@@ -0,0 +1 @@
+Add poetry entrypoint for running sydent.
```

Comment:
I think this is meant to replicate setuptools' entrypoints, but let me double check that

Reply:
Poetry's venv contains `sydent` and others as executables:

```
$ ls $(poetry env info --path)/bin
activate           black          dmypy                   mypyc       pkginfo      python3       rst2odt_prepstyles.py  stubgen              trial       wheel3.10
activate.csh       blackd         flake8                  netaddr     __pycache__  python3.10    rst2odt.py             stubtest             twine
activate.fish      black-primer   isort                   normalizer  pycodestyle  rst2html4.py  rst2pseudoxml.py       sydent               twist
activate.nu        cftp           isort-identify-imports  pip         pyflakes     rst2html5.py  rst2s5.py              sydent-casefold-db   twistd
activate.ps1       ckeygen        keyring                 pip3        pygmentize   rst2html.py   rst2xetex.py           sydent-generate-key  wheel
activate_this.py   conch          mailmail                pip-3.10    pyhtmlizer   rst2latex.py  rst2xml.py             tkconch              wheel3
automat-visualize  deactivate.nu  mypy                    pip3.10     python       rst2man.py    rstpep2html.py         tqdm                 wheel-3.10
```

I'm pretty sure that if you install the wheel this will end up creating a bin script too.